esphome/components/deep_sleep/deep_sleep_esp32.cpp

```diff
@@ -59,7 +59,7 @@ bool DeepSleepComponent::prepare_to_sleep_() {
     // Defer deep sleep until inactive
     if (!this->next_enter_deep_sleep_) {
       this->status_set_warning();
-      ESP_LOGW(TAG, "Waiting wakeup pin state change to enter deep sleep...");
+      ESP_LOGW(TAG, "Waiting wakeup pin state change...");
```

Comment:
The ellipsis is largely unnecessary (but we have it everywhere...)
```suggestion
      ESP_LOGW(TAG, "Waiting for wakeup pin state change");
```